test/scripts/block-utils.test.js

```diff
@@ -104,20 +85,20 @@ describe('Sections and blocks', () => {
 
   it('Updates section status', async () => {
     blockUtils.updateSectionsStatus(document.querySelector('main'));
-    document.querySelectorAll('main .section').forEach(($section) => {
+    document.querySelectorAll('main > div').forEach(($section) => {
```

Comment:
Are these test changes indicative of breaking changes?

Reply:
Not that I could identify so far. I was trying to decouple the loading status logic from the block/section decoration… but it's too tightly linked so will likely revert and make the decorator plugin "mandatory"